doc/src/site/sphinx/ml/sw_drf.rst

```diff
@@ -31,17 +39,33 @@ and :ref:`model_details_H2ODRFMOJOModel`.
         .. code:: scala
 
             import org.apache.spark.SparkFiles
-            spark.sparkContext.addFile("https://raw.githubusercontent.com/h2oai/sparkling-water/master/examples/smalldata/prostate/prostate.csv")
-            val rawSparkDF = spark.read.option("header", "true").option("inferSchema", "true").csv(SparkFiles.get("prostate.csv"))
-            val sparkDF = rawSparkDF.withColumn("CAPSULE", $"CAPSULE" cast "string")
+            val datasetUrl = "https://raw.githubusercontent.com/h2oai/sparkling-water/master/examples/smalldata/cars_20mpg.csv"
+            spark.sparkContext.addFile(datasetUrl) //for example purposes, on a real cluster it's better to load directly from distributed storage
+            val sparkDF = spark.read.option("header", "true").option("inferSchema", "true").csv(SparkFiles.get("prostate.csv"))
             val Array(trainingDF, testingDF) = sparkDF.randomSplit(Array(0.8, 0.2))
 
-        Train the model. You can configure all the available DRF arguments using provided setters, such as the label column.
+        Set the predictors and response columns
+
+        .. code:: scala
+
+            val predictors = Array("displacement", "power", "weight", "acceleration", "year")
+            val response = "economy_20mpg"
+
+        Build and train the model. You can configure all the available DRF arguments using provided setters, such as the label column.
 
         .. code:: scala
 
             import ai.h2o.sparkling.ml.algos.H2ODRF
-            val estimator = new H2ODRF().setLabelCol("CAPSULE")
+            val estimator = new H2ODRF()
+                  .setNtrees(10)
+                  .setMaxDepth(5)
+                  .setMinRows(10)
+                  .setCalibrateModel(true)
+                  .setCalibrationDataFrame(testingDF)
+                  .setBinomialDoubleTrees(true)
+                  .setFeaturesCols(predictors)
+                  .setLabelCol(response)
+                  .setColumnsToCategorical(response) //set the response as a factor
```

Comment:
I would also mention in the comment that casting column to the categorical will lead to the creation of a classification model.

Reply:
below is the standard note:
>"By default, the ``H2ODRF`` algorithm distinguishes between a classification and regression ..."
I added "please see the comment below" - is that ok?